defaults/networking-defaults.yaml

```diff
@@ -1,72 +1,90 @@
 # AUTO_GENERATED by scope3_methodology/cli/compute_defaults.py
 defaults:
   fixed:
-    conventional_model_generic_kwh_per_gb: 0.030000000
-    power_model_constant_watt: 9.550000000
+    conventional_model_generic_kwh_per_gb: 0.006500000
+    power_model_constant_watt: 1.500000000
```

Comment:
@bokelley Updated the kwh per gb and watt based on https://github.com/scope3data/methodology/issues/112#issuecomment-1531139434 Is this correct? Or should the watt remain the same?

Reply:
Unless you have a reason to change it I think the watts remain the same? Was there a reference to that in the docs?

Reply:
So the way I read that doc was for the power model for fixed that it was `1.5 W + 0.03 w/mbps` And since we have 0.03 set as the `power_model_variable_watt_per_mbps` that's why I changed watts here.